30.5, widoki i zdarzenia

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE STATS (
+STAT_ID INT(11) AUTO_INCREMENT,
+STAT_DATE DATETIME NOT NULL,
+STAT VARCHAR(20) NOT NULL,
+VALUE INT(11) NOT NULL,
+PRIMARY KEY (STAT_ID)
+);
+
+
+CREATE VIEW BESTSELLERS_COUNT AS
+SELECT * FROM books
+WHERE BESTSELLER = TRUE;
+
+
+CREATE VIEW BESTSELLERS_COUNT (BESTSELLERS_COUNT) AS
```

Comment:
dwa takie same widoki z rożną definicją?

Reply:
CREATE VIEW BESTSELLERS_COUNT AS
SELECT * FROM books
WHERE BESTSELLER = TRUE

ta struktura zawiera błąd, nie usunąłem, przepraszam. Właściwa to 
 CREATE VIEW BESTSELLERS_COUNT AS
SELECT Count(*) FROM books
WHERE BESTSELLER = TRUE